.github/workflows/ci.yaml

```diff
@@ -10,6 +10,10 @@ on:
 permissions:
   contents: read
 
+concurrency:
+  group: ${{ github.head_ref }}
```

Comment:
this value is only known for pull requests and crashes in merge queue for example

Reply:
Fixed